rotkehlchen/chain/ethereum/interfaces/balances.py

```diff
@@ -186,20 +150,56 @@ def _query_gauges_balances(
 
         return balances
 
+    def _get_staking_contract_balances(
```

Comment:
I am not sure if for all the protocols that use this, this will detect out of gauges, deposited tokens. Like curve/convex. This is something yet to check.